spark-orientdb-connector/src/test/scala/com/metreta/spark/orientdb/connector/ClassJsonRDDFunctionsSpec.scala

```diff
@@ -0,0 +1,120 @@
+package com.metreta.spark.orientdb.connector
+
+import com.metreta.spark.orientdb.connector.api.OrientDBConnector
+import com.metreta.spark.orientdb.connector.utils.BaseOrientDbFlatSpec
+import com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx
+import com.orientechnologies.orient.core.db.record.{OTrackedList, OTrackedMap}
+import com.orientechnologies.orient.core.metadata.schema.{OClass, OSchema}
+import org.apache.spark.rdd.RDD
+
+import scala.Vector
+import scala.collection.JavaConversions.asScalaIterator
+
+class ClassJsonRDDFunctionsSpec extends BaseOrientDbFlatSpec {
+
+  val dbname = "/tmp/databases/test/ClassJsonRDDFunctionsSpec"
+
+  var rddTest: RDD[String] = null
+  var database: ODatabaseDocumentTx = null
+
+  override def beforeAll(): Unit = {
+    defaultSparkConf.set("spark.orientdb.dbname", dbname)
+    initSparkConf(defaultSparkConf)
+    rddTest = createRDD
+    database = createDB
+    rddTest.saveJsonToOrient("class_json_ins_test")
```

Comment:
Agreed - this is exactly how they've done it in their test cases in `ClassRDDFunctionsSpec`.
I suppose I'm perpetuating the error by sticking to their flawed pattern.
I'll move it.